src/main/java/com/snowflake/kafka/connector/internal/streaming/SnowflakeSinkServiceV2.java

```diff
@@ -399,7 +399,14 @@ public void close(Collection<TopicPartition> partitions) {
   }
 
   @Override
-  public void setIsStoppedToTrue() {}
+  public void stop() {
+    try {
+      streamingIngestClient.close();
+    } catch (Exception e) {
+      LOGGER.warn("Could not close streaming ingest client");
+      throw SnowflakeErrors.ERROR_3009.getException(e);
```

Comment:
What's the connector's behavior if we throw an exception here? Will the connector stop? Or the task will die and restart?

Reply:
Good point.
I don't know what will happen with the connector (no source code available and docs remain silent about it) but it may cause some problems. The task is being stopped in this method so throwing an exception does not change much. I will leave only the warn log here.